### ISSUE TITLE
details element giving false positive for no-noninteractive-element-interactions

### ISSUE THREAD

**Issue by anthony-dandrea** [jsx-eslint/eslint-plugin-jsx-a11y#582]
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details

I'm getting what I perceive to be a false positive for `jsx-a11y/no-noninteractive-element-interactions` . This is in fact an interactive element that can be toggled open and closed natively. I'm adding listeners to the element in order to provide fallback support for IE/Edge that do not support it.

```jsx
<details
  onKeyDown={this.handleKey}
  onClick={this.toggle}
/>
```

**Comment by jessebeach** [comment]
@sh0ji is correct, this is not a false positive. The details element maps to a `group` role, which is non-interactive. The `summary` element should be considered interactive, though. We'll fix this in #656 

https://w3c.github.io/html-aria/#el-details
https://w3c.github.io/html-aria/#el-summary
